src/Tasks/Microsoft.NET.Build.Tasks.UnitTests/GivenAConflictResolver.cs

```diff
@@ -0,0 +1,435 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using FluentAssertions;
+using Microsoft.Build.Framework;
+using Microsoft.Extensions.DependencyModel;
+using Xunit;
+using Microsoft.NET.Build.Tasks.ConflictResolution;
+using System.Linq;
+
+namespace Microsoft.NET.Build.Tasks.UnitTests
+{
+    public class GivenAConflictResolver
+    {
+        //      - Different keys (same metadata) should not result in a conflict
```

Comment:
These comments were from a list of scenarios to cover I wrote before writing the tests.  I think the test names are descriptive enough so I can just delete the comments.